bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/TeaserImpl.java

```diff
@@ -81,7 +81,8 @@ protected void initImage() {
         overriddenImageResourceProperties.put(Teaser.PN_ACTIONS_ENABLED, Boolean.valueOf(actionsEnabled).toString());
         overriddenImageResourceProperties.put(PN_ID, String.join(ID_SEPARATOR, this.getId(), IMAGE_ID_PREFIX));
 
-        if (StringUtils.isNotEmpty(getTitle()) || getTeaserActions().size() > 0) {
+        // Only allow image to render links if all other elements are empty
+        if (!(StringUtils.isAllBlank(getTitle(), getPretitle(), getDescription()) && getTeaserActions().isEmpty())) {
```

Comment:
We may need '||' here instead of '&&" as before.

Reply:
I double-checked and I think this is correct:
<img width="1276" alt="image" src="https://github.com/user-attachments/assets/0853ae42-dca7-4727-b151-8df7540922cd" />

Reply:
I will prepare some test content and an IT nevertheless